alerting/src/test/kotlin/org/opensearch/alerting/DocumentMonitorRunnerIT.kt

```diff
@@ -495,6 +500,375 @@ class DocumentMonitorRunnerIT : AlertingRestTestCase() {
         assertEquals("Findings saved for test monitor expected 14, 51 and 10", 3, foundFindings.size)
     }
 
+    fun `test execute monitor with indices having fields with same name but different data types`() {
```

Comment:
can we plz add asserts on the expected mappings and content of the query index. that would be the essence of these tests and help us visualize the changes made in this PR

Reply:
asserts added on actual queries stored in query index.